ts/src/profiler.ts

```diff
@@ -13,47 +13,299 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import * as delay from 'delay';
+import * as http from 'http';
 import * as path from 'path';
+import * as pify from 'pify';
+import * as zlib from 'zlib';
 
-import {AuthenticationConfig, Common, Logger, Service, ServiceConfig} from '../third_party/types/common-types';
-
-import {Config, ProfilerConfig} from './config';
+import {perftools} from '../../proto/profile';
+import {AuthenticationConfig, Common, Logger, Service, ServiceConfig, ServiceObject, ServiceObjectConfig} from '../third_party/types/common-types';
```

Comment:
overlong line. Use `gts fix`.

Reply:
I'll fix this, but thought I'd mention I just updated gts (and gts fix has caught other overlong lines),  but gts fix did not change this line.

Reply:
After fixing this, the gts check failed, so I left it as it is.